packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -112,9 +131,13 @@ export function ListViewItem(props) {
               'focus-ring': isFocusVisible,
               'is-hovered': isHovered,
               'is-selected': isSelected,
-              'is-previous-selected': state.selectionManager.isSelected(item.prevKey),
+              'is-next-selected': state.selectionManager.isSelected(item.nextKey),
+              'is-prev-selected': state.selectionManager.isSelected(item.prevKey),
               'react-spectrum-ListViewItem--highlightSelection': state.selectionManager.selectionBehavior === 'replace' && (isSelected || state.selectionManager.isSelected(item.nextKey)),
-              'react-spectrum-ListViewItem--draggable': isDraggable
+              'react-spectrum-ListViewItem--draggable': isDraggable,
+              'react-spectrum-ListViewItem--firstRow': isFirstRow,
+              'react-spectrum-ListViewItem--lastRow': isLastRow,
+              'react-spectrum-ListViewItem--roundedCorners': shouldRoundCorners
```

Comment:
Name sucks, I would've like to apply a `react-spectrum-ListView--scrollable` classname to the ListView itself but the `layout.getContentSize()` doesn't always return the correct content size on ListView render. I'm not 100% sure why, still need to dig more, but from what I can tell it maybe due to `buildCollection` in the layout setting the content size in an async manner due to Virtualizer stuff